src/unix/pthread-fixes.c

```diff
@@ -39,19 +39,23 @@
 
 int uv__pthread_sigmask(int how, const sigset_t* set, sigset_t* oset) {
   static int workaround;
+  int errno;
```

Comment:
Sure but global errno is not needed within the stack/funtion. Could you please explain the shadowing part?